### PR TITLE
Drop support for PyTorch 1.7

### LINKED ISSUE #300: [RFC] Deprecating PyTorch 1.7
In general, we are supporting the most recent 2 releases of PyTorch. As PyTorch 1.9 is out on June 16th, we are now thinking about dropping support of PyTorch 1.7. Please feel free to leave a comment here if you have any concerns. Thanks!

### PR DESCRIPTION
**Pull request by kmaehashi**
Closes #300.

### CODE REVIEW

File: .flexci/windows/test.ps1 [hunk 1]
```diff
@@ -9,38 +9,17 @@ $ErrorActionPreference = "Stop"
...
+    ActivatePython 3.7
     RunOrDie python -m pip install -U pip setuptools
     RunOrDie python -m pip install torch==1.8.* torchvision==0.9.* -f https://download.pytorch.org/whl/cu111/torch_stable.html
 
 } elseif ($test -eq "torch19") {
-    # PyTorch 1.9 + Python 3.9
+    # PyTorch 1.9 + Python 3.8
     ActivateCUDA 11.1
-    ActivatePython 3.9
+    ActivatePython 3.8
```
Comment (asi1024) [inline .flexci/windows/test.ps1:22]: Is this fix intended?

Comment (kmaehashi) [reply .flexci/windows/test.ps1:22]: Yes, as discussed previously this is intended to cover all Python versions.

**Comment by kmaehashi** [comment]
Please regenerate docker image and trigger the test before merging :pray:

**Comment by kmaehashi** [comment]
Updated the webhook

**Pull request merged**